pkg/storaged/storage.scss

```diff
@@ -334,6 +334,10 @@ a.disabled {
     background: var(--pf-v5-global--palette--black-150);
     box-shadow: var(--pf-v5-global--BoxShadow--md);
     cursor: pointer;
+
+    .pf-v5-theme-dark & {
+	background: var(--ct-color-list-hover-bg);
```

Comment:
`--ct-color-list-hover-bg` is the correct value also for light mode, let's just replace `--pf-v5-global--palette--black-150` with it above, no need to have a special case for dark mode.